app/assets/javascripts/controllers/automation_manager/automation_manager_form_controller.js

```diff
@@ -1,20 +1,17 @@
-ManageIQ.angular.app.controller('automationManagerFormController', ['$http', '$scope', 'automationManagerFormId', 'miqService', function($http, $scope, automationManagerFormId, miqService) {
+ManageIQ.angular.app.controller('automationManagerFormController', ['$http', '$scope', 'automationManagerFormId', 'miqService', 'configurationManagerService', function($http, $scope, automationManagerFormId, miqService, configurationManagerService) {
   var vm = this;
 
-  vm.automationManagerModel = {
-    name: '',
-    url: '',
-    zone: '',
-    verify_ssl: '',
-    log_userid: '',
-    log_password: '',
-  };
+  vm.automationManagerModel = configurationManagerService.managerModel;
```

Comment:
I do not anticipate issues with the scenario that you have described above, but a `copy` won't hurt.